homeassistant/components/surepetcare/binary_sensor.py

```diff
@@ -149,15 +119,13 @@ class DeviceConnectivity(SurePetcareBinarySensor):
 
     def __init__(
         self,
-        _id: int,
-        coordinator: DataUpdateCoordinator,
+        surepetcare_id: int,
+        coordinator: SurePetcareDataCoordinator,
     ) -> None:
         """Initialize a Sure Petcare Device."""
-        super().__init__(_id, coordinator)
-        self._attr_name = f"{self.name}_connectivity"
-        self._attr_unique_id = (
-            f"{self.coordinator.data[self._id].household_id}-{self._id}-connectivity"
-        )
+        super().__init__(surepetcare_id, coordinator)
+        self._attr_name = f"{self._device_name} Connectivity"
```

Comment:
I think the entity name change could be a breaking change.

Reply:
If the unique_id stays the same, we can change the default name without breaking it for users.

Reply:
Thanks for the clarification.

Reply:
The unique_id is unchanged